src/features/featureincludeallnetworks.h

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef FEATURE_INCLUDEALLNETWORKS_H
+#define FEATURE_INCLUDEALLNETWORKS_H
+
+#include "models/feature.h"
+
+constexpr const char* FEATURE_INCLUDEALLNETWORKS = "includeAllNetworks";
+
+class FeatureIncludeAllNetworks : public Feature {
+ public:
+  FeatureIncludeAllNetworks()
+      : Feature(FEATURE_INCLUDEALLNETWORKS, "Include All Networks",
+                false,               // Is Major Feature
+                L18nStrings::Empty,  // Display name
+                L18nStrings::Empty,  // Description
+                L18nStrings::Empty,  // LongDescr
+                "",                  // ImagePath
+                "",                  // IconPath
+                "",                  // link URL
+                "2.8",               // released
+                true                // Can be enabled in devmode
+        ) {}
+
+  bool checkSupportCallback() const override {
+#if defined(MVPN_IOS) 
```

Comment:
Let's include this feature only on iOS. And to enforce it, At the top of this file:

```
#ifndef MVPN_IOS
#error "This feature should be exposed only to IOS - or a better error message here...
#endif
```

And here you can simply say: `return true;`